lib/aruba/generators/script_file.rb

```diff
@@ -0,0 +1,37 @@
+# Aruba
+module Aruba
+  # ScriptFile
+  class ScriptFile
+    def initialize(opts = {})
+      @path        = opts[:path]
+      @content     = opts[:content]
+      @interpreter = opts[:interpreter]
+    end
+
+    def call
+      script_content = @content.start_with? '#!' ? @content : "#{header}\n#{@content}"
```

Comment:
Please move to `def header`:

~~~ruby
if script_starts_with_shebang?
  ''
elsif interpreter_is_absolute_path?
# [...]
~~~

~~~ruby
private

# [...]
def script_starts_with_shebang?
  @content.start_with? '#!' 
end
~~~

And also move the "\n" to the `#format`-methods. Then we can just `#{header}#{content}`.
~~